modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -57,7 +57,14 @@ Runtime Fabric enables you to specify the severity level of messages that are wr
 Log levels are specified per Mule application or API proxy during deployment.
 
 [WARNING]
-Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
+
+* Because log levels are specified when deploying a Mule application or an API proxy, you cannot change the log levels after deployment.
+* If you are using Mule runtime engine version 4.3.0, you cannot change the log level using the `log4j2.xml` configuration file. To change the Runtime Fabric log level when using Mule runtime engine version 4.3.0, you must use Anypoint Runtime Manager to change the following property:
+```
+logging.level.<PACKAGE>=LEVEL
+``` 
+For example, if you use Runtime Manager to add the 
+`logging.level.org.mule.service.http.impl.service.HttpMessageLogger=DEBUG` property, `DEBUG` level log information is then retrieved from `HttpMessageLogger`.
```

Comment:
For example, if you use Runtime Manager to add the 
`logging.level.org.mule.service.http.impl.service.HttpMessageLogger=DEBUG` property, `DEBUG` level log information is emitted from `HttpMessageLogger`

Reply:
Updated this - thank you!